File: Luento3/W3/W3_T1/person.py
class Person:
    def __init__(self, fname: str, lname: str, age: int) -> None:
        self.first_name = fname
        self.last_name = lname
        self.__age = age

    def getAge(self) -> int:
        return self.__age
    
    def ageUp(self) -> int:
        self.__age += 1

    def getFullname(self) -> str:
        return f"{self.first_name} {self.last_name}"
    
    def printFullname(self) -> None:
        print(self.getfullname())

File: Luento3/W3/W3_T6/person.dart
class Person {
  String first_name;
  String last_name;
  int _age;

  Person(this.first_name, this.last_name, this._age);

  int getAge() {
    return _age;
  }

  void ageUp() {
    _age += 1;
  }

  String getFullname() {
    return "$first_name $last_name";
  }

  void printFullname() {
    print(getFullname());
  }
}
